src/current/v24.3/create-schedule-for-backup.md

```diff
@@ -9,12 +9,6 @@ docs_area: reference.sql
 
 For more information about creating, managing, monitoring, and restoring from a scheduled backup, see [Manage a Backup Schedule]({% link {{ page.version.version }}/manage-a-backup-schedule.md %}).
```

Comment:
There is also a reference to Enterprise licenses in the Parameters table under `FULL BACKUP crontab`.

Reply:
"If you do not have an Enterprise license then you can only take full backups."

Reply:
Ah thanks! Updated to remove it